licences-specs/src/test/groovy/uk/gov/justice/digital/hmpps/licences/specs/common/WebsiteSpec.groovy

```diff
@@ -63,11 +63,7 @@ class WebsiteSpec extends GebReportingSpec {
 
     then: 'I see health status OK'
 
-    def json = (
-      driver.pageSource
-        - ' xmlns="http://www.w3.org/1999/xhtml"'
-        - '<html><head></head><body><pre style="word-wrap: break-word; white-space: pre-wrap;">'
-    ) - '</pre></body></html>'
+    def json = (driver.pageSource.replaceAll("\\<.*?>", "")) - '</pre></body></html>'
```

Comment:
technically shouldnt need the`'</pre></body></html>'` either

Reply:
Ah cool, will make this update now quickly